api/helpers/common_helper.py

```diff
@@ -12,3 +12,7 @@ def fetch_by_key(key, data):
     if key in data:
         return data[key]
     return None
+
+
+def is_string_contains_key(key, string):
```

Comment:
1. Please add documentation on the purpose of the function.
2. Rename the function to something like `has_substring` or `has_key` or `is_key_present`.

Reply:
This is not relevant any more as the function definition has changes now. Created a new one with function documentation.